backend.py

```diff
@@ -20,13 +20,15 @@ def __init__(self, direction, coordinates, name):
         self.direction = direction
         self.coordinates = coordinates
         self.start_coordinates = coordinates
-        self.program = []
+        self.program = [None, None, None, None, None]
         self.lives = 3
         self.flags = 0
         self.damages = 0
         self.power_down = False
         self.name = name
         self.selection_confirmed = False
+        self.unblocked_cards = 5
```

Comment:
Počet blokovaných karet by se měl dát spočítat z ostatních atributů, podobně jako se `inactive` zjišťuje podle `coordinates`. Je to tak?

Reply:
myslíš neblokovaných. Jo, je to `MAX_CARD_COUNT - self.damages` ale pouze v případě že je počet zranění větší než 4, jinak je to vždy 5 tak udělám metodu a bude se počítat :)